src/components/molecules/ShareModal.tsx

```diff
@@ -24,6 +24,7 @@ import { tags } from "@src/state/api/tags";
 import DefaultSpinner from "@src/components/atoms/DefaultSpinner";
 import PrimaryButton from "@src/components/atoms/PrimaryButton";
 import { useUser } from "@src/state/user/hooks";
+import { FlexRowAligned } from "../styled";
```

Comment:
should prob not use styled components moving forward, opting for tailwind classes instead